components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -8875,6 +8875,8 @@ def getAllRenderingDefs(self, eid=-1):
                     # 'reverseIntensity' is deprecated. Use 'inverted'
                     'inverted': reverse,
                     'reverseIntensity': reverse,
+                    'family': unwrap(w.getFamily().getValue()),
+                    'coefficient': unwrap(w.getCoefficient()),
```

Comment:
shouldn't be ``w.getCoefficient().getValue()``?

Reply:
That will throw an exception if the value is `None`. Using `unwrap()` with `w.getCoefficient()` is correct.

Reply:
thanks